src/viur/core/render/html/env/viur.py

```diff
@@ -725,3 +728,16 @@ def seoUrlForEntry(render: Render, *args, **kwargs):
 @jinjaGlobalFunction
 def seoUrlToFunction(render: Render, *args, **kwargs):
     return utils.seoUrlToFunction(*args, **kwargs)
+
+
+@jinjaGlobalFunction
+def data_to_qrcode(render: Render, data: str, *args, **kwargs) -> str:
```

Comment:
`qrcode` would be a nice, short name for this function. It should be clear, that data is turned into a QR-code.
Any other args/kwargs are invalid.
```suggestion
def qrcode(render: Render, data: str) -> str:
```